Dockerfile

```
@@ -1,8 +1,16 @@
-FROM alpine:3.10
+FROM ballerina/ballerina:2201.2.2
 
 # install packages required to run the tests
+USER root
 RUN apk add --no-cache jq coreutils
 
-WORKDIR /opt/test-runner
-COPY . .
-ENTRYPOINT ["/opt/test-runner/bin/run.sh"]
+ENV wd /opt/test-runner
+RUN mkdir -p ${wd}/bin
+WORKDIR ${wd}
```

I don't think you need to create the `bin` directory

```suggestion
WORKDIR /opt/test-runner
```

---

Sounds good. I found this sample in the rust-test-runner.